vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "System.Management.Automation": "6.1.7601.17515",
```

Comment:
but `DeployTest` uses powershell scripts that themselves use WMI, so not really. Also looked at that test class, and all tests are marked as `Skip="FIX me"`, so let's not add this yet for some tests that don't even run.